pkg/api/mlflow/service/artifact/service_test.go

```diff
@@ -15,55 +15,63 @@ import (
 )
 
 func TestService_ListArtifacts_Ok(t *testing.T) {
-	artifactStorage := storage.MockProvider{}
+	artifactStorage := storage.MockArtifactStorageProvider{}
 	artifactStorage.On(
-		"List", "/artifact/uri", "",
+		"List", context.TODO(), "/artifact/uri", "",
```

Comment:
I'd change to `context.Background()` for tests, since there is no intention to do anything else.